app/javascript/components/message.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+const GET_RANDOM_MSG_REQUEST = 'GET_RANDOM_MSG_REQUEST';
+const GET_RANDOM_MSG_SUCCSESS = 'GET_RANDOM_MSG_SUCCSESS';
+
+const getRandomMsgSuccess = (json) => ({
+  type: GET_RANDOM_MSG_SUCCSESS,
+  json,
+});
+
+const getRandomMsg = () => (dispacth) => {
+  dispacth({ type: GET_RANDOM_MSG_REQUEST });
+  return fetch('api/v1/randomMessage')
```

Comment:
- [ ] When I change the `api` to the rails local-host address. It doesn't work. I will suggest you make the above corrections and ensure the react frontend works as detailed in the requirements.

Reply:
- [ ] This is still not working. I will suggest you fix it.